perf/common.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+exports.createMsg = function(msgType, sz) {
+    var buf;
```

Comment:
eh, for tests/perf I don't care as much, and I haven't finished the JS formatting yet.